Packs/CortexVulnerabilityManagement/Scripts/ContextSetup/README.md

```diff
@@ -0,0 +1,54 @@
+Automation used to more easily populate XSOAR context.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Instead of a value you can enter `TIMESTAMP` to get the current timestamp in ISO format. For example:
```

Comment:
```suggestion
Automation used to more easily populate Cortex XSOAR context. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Instead of a value you can enter `TIMESTAMP` to get the current timestamp in ISO format. For example:
```